devel/mercurial/Portfile

```diff
@@ -7,7 +7,7 @@ PortGroup           bitbucket 1.0
 name                mercurial
 # don't forget to update dependents for mercurial:
```

Comment:
Please read the comments :)

Reply:
Yes, you are right. I did the updated port one month ago and I commit this during a clean without re-check. Sorry for the mistake. So, probably, I can close this and follow-up your PR.